tests/validation_tests/storage_tuple.solution.json

```diff
@@ -2,8 +2,8 @@
   "data": [
     {
       "predicate_to_solve": {
-        "contract": "B209AF469914AA6DE8ECA843A31351712FFA491AAE71CE2CD9F935597EBB70E9",
-        "predicate": "3D7D302EE1E82A25B262B6F6F728E0A66E75A61D0C6A78AB5AD913796AE049A5"
+        "contract": "6F7917A688D243501B23C847DF70A21C32AF5868E3CEBD96A3D9056021D89CA1",
+        "predicate": "4B1CF45F80623E58BD0C7B90989B42F9BBB0C054D33C8181170C22E078DE3C2C"
```

Comment:
Any idea why I'm not able to hash the proper addresses for the contract and predicates in `validation_e2e` tests?

I've used the local `pint build` to get the hash of the bytecode for `storage_tuple.pnt`, and I'm still getting the error `Error: solution for test validation_tests/storage_tuple.pnt does not solve any predicates in the test!`

I jumped back to the `master` branch and I'm unable to reproduce the original hashes too. Even weirder, I get 
```
"contract": "6F7917A688D243501B23C847DF70A21C32AF5868E3CEBD96A3D9056021D89CA1",
"predicate": "4B1CF45F80623E58BD0C7B90989B42F9BBB0C054D33C8181170C22E078DE3C2C"
```
for both this branch and `master`.

Reply:
I pushed a fix for `storage_tuple` that updates the addresses. You need to do the same for the other tests one by one. Make sure you're compiling `pint` and using the build. When you run the tests, the binary `pint` is not actually updated. You need to run `cargo build` separately.

Reply:
>... You need to run cargo build separately.

This was the issue. 🤦 Thank you sir!